garrysmod/gamemodes/sandbox/entities/entities/gmod_cameraprop.lua

```diff
@@ -213,22 +194,22 @@ end
 function ENT:CanTool( ply, trace, mode )
 
 	if ( self:GetMoveType() == MOVETYPE_NONE ) then return false end
-	
+
 	return true
 
 end
 
-function ENT:Draw()	
+function ENT:Draw()
 
-	if ( GetConVarNumber( "cl_drawcameras" ) == 0 ) then return end
+	if ( GetConVar( "cl_drawcameras" ):GetInt() == 0 ) then return end

```

Comment:
The whole point of GetConVarNumber being deprecated is to encourage you to reuse the value returned by GetConVar.